mlir/include/mlir/Dialect/Bufferization/Transforms/Passes.h

```diff
@@ -147,7 +147,7 @@ std::unique_ptr<Pass> createBufferLoopHoistingPass();
 
 // Options struct for BufferResultsToOutParams pass.
 // Note: defined only here, not in tablegen.
-struct BufferResultsToOutParamsOptions {
+struct BufferResultsToOutParamsOpts {
```

Comment:
Is there a specific reason to diverge from the upstream name of the parameter options?